docs/content/concepts/ops-jobs-graphs/op-events.mdx

```diff
@@ -127,17 +127,19 @@ from dagster import op, AssetMaterialization, Output, EventMetadata
 def my_metadata_materialization_op(context):
     df = read_df()
     remote_storage_path = persist_to_storage(df)
-    yield AssetMaterialization(
```

Comment:
It's worth searching for instances of "yield" in this entire document and seeing if they need to be changed.

Reply:
Also, I think it would still be useful to include an example of yielding, so that users who have been using it up to this point can discover that it's still supported.